### ISSUE TITLE
Are there other more generic applications of sadde-free Newton method?

### ISSUE THREAD

**Issue by lih2app** [dave-fernandes/SaddleFreeOptimizer#2]
Hi,

I am wondering if there are other applications of this method other than on neural networks. Perhaps like finding the global minimum of a multidimensional function?

Thank you,
Li

**Comment by dave-fernandes** [comment]
This method would be applicable to minimizing other functions than neural networks. However, like gradient descent, it will find a local minimum. It has no way to know when that is not a global minimum.
